config/alphagov.yml

```diff
@@ -96,23 +96,12 @@ govuk-pub-workflow:
   members:
     - 1pretz1
     - alex-ju
-    - benhazell
     - benthorner
     - emmabeynon
-    - erino
     - kevindew
-    - kgarwood
-    - soniaturcotte
 
   channel:
-    "#govuk-pub-workflow"
-
-  exclude_titles:
-    - "[DO NOT MERGE]"
-    - "Don't merge"
-    - "DON'T MERGE"
-    - "DO NOT MERGE"
-    - "WIP"
```

Comment:
We do benefit from some of these - WIP and DO NOT MERGE in particular. These could/should be replaced by draft PR's but this project doesn't acknowledge these yet.

Reply:
Ah, sorry about that. I thought the point of Seal is to notify us about pending PRs as a form of continuous review. If we're concerned about the spam, then let's continue to exclude, but shall we limit it to the two you mentioned, instead of all these variants?

Reply:
Yup, that sounds good.

If you want to be extra awesome too you could remove draft PR's from all results. From a little bit of experimenting it looks like we can do that with a change to: https://github.com/alphagov/seal/blob/d15e9a9738b5b8acdd29901a53e966f454413538/lib/github_fetcher.rb#L59 to have `-is:draft` included as per: https://github.com/search?q=state%3Aopen+author%3Aalex-ju+is%3Apr+-is%3Adraft

Reply:
Sure, updated - I was thinking about the draft one, but at the time my thoughts were we shouldn't ignore them. I might do it later :-).

Reply:
Nice one 👍 

As these alerts are meant to be call to actions to review a PR to get it merged then I wouldn't consider drafts part of the criteria of the seal reminders. There's more just an onus on the author to remove the draft status of the PR.